scripts/deployment/main.py

```diff
@@ -264,8 +264,8 @@ def _margin_gas_price_strategy(web3: Web3, transaction_params: TxParams) -> Wei:
     resolver = deploy_contract(web3_l1, "Resolver")
 
     deployment_data: dict = {"beamer_commit": get_commit_id(), "deployer": account.address}
-    deployment_data["L1"] = collect_contracts_info({"Resolver": resolver})
-    deployment_data["L2"] = {}
+    deployment_data["base_chain"] = collect_contracts_info({"Resolver": resolver})
```

Comment:
may have some problems in relayer/fe side but lets see

Reply:
With `base-chain` as a property name (instead of `base_chain`) we would have to access it in the TS/JS code as `deployment['base-chain']` instead of `deployment.base_chain`.
That will be the only thing affected by this change.

Reply:
We rarely access properties with `['']` in TS/JS but if we want to be consistent with naming (we should) then i am fine with this small exception.

Reply:
Why don't we just use `base_chain` as a property name inside the deployment config files ??
If we want to be consistent with property name casing than we should apply this change.

Reply:
this was the same thinking when i first answer it like this :D

Reply:
Yeah, ok, if it makes the fronted part easier, let's leave this as-is and after this lands we can convert the deployment config files to be in sync with this.

Reply:
So its not forgotten: https://github.com/beamer-bridge/beamer/issues/1619